### PR TITLE
 fix header

### PR DESCRIPTION
**Pull request by fullstop125**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+header {
+  color: black;
+  margin: 10px;
+  text-align: center;
+  font-size: 1.8em;
+  font-weight: thin;
+}
+
```
Comment (fullstop125) [inline css/style.css:28]: fix linters

**Review by fullstop125 — COMMENTED** [review]
abscc

File: index.html [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+      </section>
+      <section class="main-section" id="Reference">
+        <header>Reference</header>
+        <article>
+           <ul>
+            <li>
+              All the documentation in this page is taken from
+              <a href="https://docs.microsoft.com/en-us/cpp/?view=msvc-170"target="_blank">MDN</a
+              
+            </li>
```
Comment (fullstop125) [inline index.html:243]: hello, I Saw “<” when expecting an attribute name. Probable cause: Missing “>” immediately before

**Review by fullstop125 — COMMENTED** [review]
Hi @fullstop125, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Git flow was correctly used




## Required Changes ♻️
- [ ] Kindly add a  descriptive pull request message, it should be a brief summary describing changes introduced in your pr.
- [ ] Saw “<” when expecting an attribute name. Probable cause: Missing “>” immediately before on line 234
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @fullstop125.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — COMMENTED** [review]
Hi @fullstop125, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Use of polite suggestions during review
✅ Use of an inline comment
✅ Mentioned reason for suggesting a change
✅ Use of emojis

## Optional suggestions
✅ You mentioned `line 234` in your review when you probably meant `line 243` 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — APPROVED** [review]

**Comment by fullstop125** [comment]
Thank for the correction @fobadara I appreciate😇😇

**Pull request closed without merging**